### PR TITLE
[ja] Localize Tightly coupled architectures for Japanese

### LINKED ISSUE #2137: [ja] Localize Tightly Coupled Architectures for Japanese
- Localize Tightly Coupled Architectures for Japanese
- https://glossary.cncf.io/tightly-coupled-architectures/

### PR DESCRIPTION
**Pull request by bashi8128**
### Describe your changes
Localize Tightly coupled architectures for Japanese

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2137

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by kaitoii11 — APPROVED** [review]

File: content/ja/tightly-coupled-architectures.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: 密結合アーキテクチャ
+status: 完了
+category: プロパティ
+tags: ["基礎", "アーキテクチャ", "プロパティ"]
+---
+
+密結合アーキテクチャは、アプリケーション構成要素の多くが互いに依存しているようなアーキテクチャを指します
+(その逆は[疎結合アーキテクチャ](/loosely-coupled-architecture/)と呼ばれます)。
```
Comment (naonishijima) [inline content/ja/tightly-coupled-architectures.md:9]: please add ja path in link like below.
/ja/loosely-coupled-architecture/

File: content/ja/tightly-coupled-architectures.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+密結合であることは、ある構成要素に対する変更がその他の構成要素に影響を与えうることを意味します。
+一般的に、密結合アーキテクチャはより疎結合なアーキテクチャと比較して簡単に実装できますが、カスケード障害と呼ばれる連鎖的な障害に対してシステムがより弱くなる可能性があります。
+また、しばしば構成要素を協調してロールアウトする必要が生じるため、開発者の生産性に対する足かせとなる可能性があります。
+
+密結合なアプリケーションアーキテクチャはかなり古典的なアプリケーション構築方法です。
+[マイクロサービス](/microservices/)開発のベストプラクティス全てには必ずしも整合しませんが、
```
Comment (naonishijima) [inline content/ja/tightly-coupled-architectures.md:15]: ditto

File: content/ja/tightly-coupled-architectures.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+密結合アーキテクチャは特定の場合に有用です。
+密結合アーキテクチャでの開発は、実装がより簡潔で早くなる傾向があり、
+[モノリシックアプリケーション](/monolithic-apps/)とよく似て初期開発サイクルを加速させる可能性があります。
```
Comment (naonishijima) [inline content/ja/tightly-coupled-architectures.md:18]: ditto

File: content/ja/tightly-coupled-architectures.md (hunk 1 above)
Comment (bashi8128) [reply content/ja/tightly-coupled-architectures.md]: Thank you for your comments.
I fixed the link in [cf90e8c](https://github.com/cncf/glossary/pull/2365/commits/cf90e8c7ec15ac133c6defe0e2a9882e4863be09).

File: content/ja/tightly-coupled-architectures.md (hunk 2 above)
Comment (bashi8128) [reply content/ja/tightly-coupled-architectures.md]: Fixed in [cf90e8c](https://github.com/cncf/glossary/pull/2365/commits/cf90e8c7ec15ac133c6defe0e2a9882e4863be09).

File: content/ja/tightly-coupled-architectures.md (hunk 3 above)
Comment (bashi8128) [reply content/ja/tightly-coupled-architectures.md]: Fixed in [cf90e8c](https://github.com/cncf/glossary/pull/2365/commits/cf90e8c7ec15ac133c6defe0e2a9882e4863be09).

**Comment by bashi8128** [comment]
マークダウン中のStatusについて、`完了`ではなく`Completed`にしておく必要がありました(Statusが`完了`だと、[この条件](https://github.com/cncf/glossary/blob/8cd55b7baf857bed85d315a75436b8f67f9b8698/assets/js/offline-search.js#L60)にひっかからないみたいで実際に表示する際サイドバーに単語が出てこない)。
そちらも修正しています。

I corrected the inappropriate document status from `完了` to `Completed` because the document becomes hidden in the sidebar tree if its status is not set to exactly `Completed`.

**Review by yuichi-nakamura — APPROVED** [review]
LGTM

**Pull request merged**